<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>160</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>160</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReplace">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Заменить &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHTML">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>120</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>HTML &gt;&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
